src/components/shared/VirtualizedScrollableGrid.js

```diff
@@ -0,0 +1,222 @@
+import { debounce } from 'underscore';
+import { FixedSizeGrid as List } from 'react-window';
+import { WindowScroller } from 'react-virtualized';
```

Comment:
I'd say that you have to import `WindowScroller` specifying the path. Otherwise the full library is bundled.

Reply:
Yes, you were right! 😬 Updated it now